infra/templates/azure-simple/variables.tf

```diff
@@ -49,6 +49,55 @@ variable "appgateway_http_listener_protocol" {
   default     = "Https"
 }
 
+# Monitoring Service
+variable "action_group_name" {
+  description = "The name of the action group."
+  type        = "string"
+  default     = "Simple Default Action Group"
+}
+
+variable "action_group_email_receiver" {
+  description = "The e-mail receiver for an alert rule resource."
+  type        = "string"
+  default     = ""
+}
+
+variable "metric_alert_criteria_namespace" {
+  description = "A monitored resource namespace with configurable metric alert criteria."
+  type        = "string"
+  default     = "Microsoft.Web/serverfarms"
+}
+
+variable "metric_alert_name" {
+  description = "The display name of a group of metric alert criteria."
+  type        = "string"
+  default     = "Simple Default Metric Alerts"
+}
+
+variable "metric_alert_criteria_name" {
+  description = "A predefined Azure resource alert monitoring rule name."
+  type        = "string"
+  default     = "CpuPercentage"
+}
+
+variable "metric_alert_criteria_aggregation" {
+  description = "The calculation used for building metric alert criteria."
+  type        = "string"
+  default     = "Average"
+}
+
+variable "metric_alert_criteria_operator" {
+  description = "A logical operator used for building metric alert criteria."
+  type        = "string"
+  default     = "GreaterThan"
+}
+
+variable "metric_alert_criteria_threshold" {
+  description = "The criteria threshold value that activates the metric alert."
+  type        = "string"
+  default     = "10"
```

Comment:
Is this a reasonable default? > 70% CPU seems alarming, but any lower is likely OK

Also what time period is this metric collected over?

Reply:
I feel defaulting our rules to a 10% CPU Average alert would result into a busy inbox for module users.

Reply:
> I feel defaulting our rules to a 10% CPU Average alert would result into a busy inbox for module users.

Default is now at 50%. High enough not to 'flood' an inbox but low enough for a demonstration that the notifications are working.

Reply:
>

> Is this a reasonable default? > 70% CPU seems alarming, but any lower is likely OK
> 
> Also what time period is this metric collected over?

I added time frequency and time period variables with descriptions and default variables at the template level. ("metric_alert_frequency"; "metric_alert_period")